kodi_addon_checker/xml_schema/matrix_binary_audiodecoder.xsd

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE schema PUBLIC "-//W3C//DTD XMLSCHEMA 200102//EN" "http://www.w3.org/2001/XMLSchema.dtd">
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
+  <xs:element name="extension">
+    <xs:complexType>
+      <xs:attribute name="point" type="xs:string" use="required"/>
+      <xs:attribute name="name" type="xs:string"/>
+      <xs:attribute name="extension" type="xs:string"/>
+      <xs:attribute name="tags" type="xs:boolean"/>
+      <xs:attribute name="library_android" type="xs:string"/>
```

Comment:
You can probably restrict this to *.so with a regex pattern whenever applied, dylib for osx and dll in windows. Maybe too much effort as we are not checking binary add-ons at the moment

Reply:
I can look about should be possible by regex.

Reply:
Have added to all binary parts a regex about, the test at home has worked.

OK:
- audiodecoder.2sf.so.12.2.0
- audiodecoder.2sf.so.12
- audiodecoder.2sf.so
- audiodecoder.blabala.aaa.2sf.so.12.2.0

ERROR (as wanted):
- audiodecoder.2sf.so.12.2
- audiodecoder.2sf.so.12.2a.0
- audiodecoder.2sf.so.12z.2.0
- audiodecoder.2sf.so.
- audiodecoders.2sf.so
- audiodecoder.2sf.sol.12.2.0